plugins/dell-dock/fu-dell-dock-i2c-ec.c

```diff
@@ -52,6 +52,18 @@ typedef enum {
 	FW_UPDATE_AUTHENTICATION_FAILED,
 } FuDellDockECFWUpdateStatus;
 
+typedef enum {
+	DOCK_MODE_CAP_TBT        = 1 << 0,
+	DOCK_MODE_CAP_DP         = 1 << 1,
+	DOCK_MODE_CAP_USB3       = 1 << 2,
+	DOCK_MODE_CAP_DOCKING    = 1 << 3,
+	DOCK_MODE_CAP_EXT_POWER  = 1 << 4,
+	DOCK_MODE_CAP_USB4       = 1 << 5,
+	DOCK_MODE_CAP_USB2       = 1 << 6,
+	DOCK_MODE_CAP_SOUCE_BACK = 1 << 7,
```

Comment:
is this a misspelling?

Reply:
it's custom that notebook is capable of sourcing back power to dock

Reply:
So it's just missing an 'r'

Reply:
Thanks for the catch! :)

Reply:
Removed for now.